model/db-common.yaml

```diff
@@ -1,22 +1,39 @@
 groups:
-  - id: attributes.db.client
+  - id: attributes.db.client.minimal
     type: attribute_group
     brief: 'Database Client attributes'
     attributes:
-      - ref: db.namespace
-        requirement_level:
-          conditionally_required: If available.
-      - ref: db.collection.name
+      - ref: db.operation.name
         requirement_level:
           conditionally_required: >
             If readily available. Otherwise, if the instrumentation library parses `db.query.text` to capture
-            `db.collection.name`, then it SHOULD be the first collection name found in the query.
-      - ref: db.operation.name
+            `db.operation.name`, then it SHOULD be the first operation name found in the query.
+      - ref: server.address
+        brief: >
+          Name of the database host.
+      - ref: server.port
+        requirement_level:
+          conditionally_required: If using a port other than the default port for this DBMS and if `server.address` is set.
+      - ref: error.type
+        requirement_level:
+          conditionally_required: If and only if the operation failed.
+        note: >
+          The `error.type` SHOULD match the error code returned by the database or the client library,
+          the canonical name of exception that occurred, or another low-cardinality error identifier.
+          Instrumentations SHOULD document the list of errors they report.
+
+  - id: attributes.db.client.all_common
```

Comment:
this looks like it's only used in one place currently, does it make sense to inline there?